src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -601,6 +629,11 @@ static void glb_destroy(grpc_exec_ctx *exec_ctx, grpc_lb_policy *pol) {
     grpc_grpclb_destroy_serverlist(glb_policy->serverlist);
   }
   gpr_mu_destroy(&glb_policy->mu);
+
+  for (size_t i = 0; i < glb_policy->num_ok_serverlist_addresses; ++i) {
```

Comment:
Looks like we've got this same code in two different places.  How about creating a `lb_addrs_destroy()` helper function to avoid the duplication?

This would probably also allow you to inline the code that's currently in `user_data_destroy()`, which would eliminate the need for that function.